src/_includes/layouts/handbook.njk

```diff
@@ -193,3 +193,20 @@ date: git Last Modified
 <!-- Syntax Highlighting CSS -->
 <link rel="preload" href="https://cdn.jsdelivr.net/npm/prismjs@1.29.0/themes/prism-tomorrow.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'" crossorigin="anonymous" referrerpolicy="no-referrer" />
 
+
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@algolia/algoliasearch-netlify-frontend@1/dist/algoliasearchNetlify.css" />
```

Comment:
Both these stylesheets can be deferred using the preload as trick: https://web.dev/defer-non-critical-css/